src/components/EditServerDetails.js

```diff
@@ -163,6 +175,13 @@ class EditServerDetails extends Component {
     );
   }
 
+  renderButtonForDropDown(addAction, buttonLabel) {
+    return (
+      <ActionButton type={'default'} clickAction={addAction} moreClass={'inline-button'}
+        displayLabel={translate(buttonLabel) + ' ...'}/>
```

Comment:
I'm confused whats going on here... I dont see any buttons on the page that have ... in the label, and I'm not sure that appending it here is the best route to go (maybe it is, I'm not sure... are there languages where the ... wouldnt be appropriate?)

Reply:
@jeremy-moffitt  this is just an existing render function for those nic-mapping and server groups button in the edit dialog, I just moved out into a function so I can hide them since adding compute nodes along with adding nic-mapping or server group is not thought through.

Reply:
ah ok, I see, since its not a change, we can worry about the translation feedback some other time (it likely wont come up)